Workbooks/Virtual Machines - Performance Analysis/Performance Counters/settings.json

```diff
@@ -2,6 +2,7 @@
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
     "name":"Performance Counters",
     "author": "Microsoft",
+    "description": "Provides CPU, Memory, Disk, and Network metrics collected by Azure Monitor for VMs (uses 'InsightsMetrics' table in Log analytics).",
```

Comment:
I think this one should have a similar description as the file above?
`Provides performance metrics collected from machines (uses 'InsightsMetrics' table in Log analytics)`

Reply:
The reason why CPU, Memory, Disk, and Network metrics are called out explicitly instead of 'performance metrics' is because Azure Monitor for VMs collects only those.